cmd/virtual-kubelet/commands/root/tracing.go

```diff
@@ -96,7 +97,14 @@ func setupZpages(ctx context.Context) {
 	if p == "" {
 		log.G(ctx).Error("Missing ZPAGES_PORT env var, cannot setup zpages endpoint")
 	}
+	listener, err := net.Listen("tcp", p)
+	if err != nil {
+		log.G(ctx).WithError(err).Error("Cannot bind to ZPAGES PORT, cannot setup listener")
+	}
 	mux := http.NewServeMux()
 	zpages.Handle(mux, "/debug")
-	http.ListenAndServe(p, mux)
+	go func() {
+		// This should never terminate, if it does, it will always terminate with an error
+		log.G(ctx).WithError(http.Serve(listener, mux)).Error("Zpages server exited")
```

Comment:
I'd rather not log an error if it's not an error.

> After Shutdown or Close, the returned error is ErrServerClosed

We should be able to check for this and only log the error if it's not that error.